src/api/public/apidocs-new/paths/source_project_name_package_name_binary_filename_attribute_name.yaml

```diff
@@ -0,0 +1,53 @@
+get:
+  summary: Get list of attributes and attribute values
+  description: |
+    Get a list of all attributes and their values or a list of attribute values when attribute is provided in parameter
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: path
+      name: package_name
+      schema:
+        type: string
+      description: Package name
+      example: "ctris"
+    - in: path
+      name: binary_filename
+      schema:
+        type: string
+      description: Binary filename
+      example: ctris-0.42.1-8.1.x86_64.rpm
+    - in: path
+      name: attribute_name
+      schema:
+        type: string
+      description: The name of the attribute with namespace
+      example: OBS:OwnerRootProjectTest
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/attributes.yaml'
+          example:
+            name: AllowSubmitToMaintenanceRelease
+            namespace: OBS
```

Comment:
As you are documenting the endpoint with and without specifying the `attribute_name`, I would give two response examples here. One with one single attribute and another one with multiple attributes.

Apart from that, the response is slightly different when a binary is involved. This is an example:

```
<attributes>
  <attribute name="ScreenShots" namespace="OBS" binary="hello_world-1-lp153.9.1.x86_64.rpm"/>
</attributes>
```

So `binary` should be included in addition to `name` and `namespace` in the schema and example.

Reply:
Done, I've included more examples